.github/dependabot.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: 2
+enable-beta-ecosystems: true
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  allow:
+  - dependency-type: "all"
+  schedule:
+    interval: weekly
+    day: "wednesday"
```

Comment:
Given that releases tend to start on a Monday, would it not be better to have this PR be made on Monday or Friday morning?

Reply:
The thinking behind Wednesday was ensuring being after patch Tuesday. But sure Monday makes sense.

Reply:
Actually, that's a good idea - hearing that I'd keep it on Wednesday, but maybe add a comment specifically calling out why. because it wasn't immediately obvious to me.